src/Domain/Fixture/FixtureCreator.php

```diff
@@ -291,18 +278,6 @@ private function fixDefaultDependencies(): void {
 
     $additions[] = "drupal/core-dev:{$this->options->getCore()}";
 
-    if ($this->shouldRequireProphecyPhpunit()) {
-      $additions[] = 'phpspec/prophecy-phpunit:^2';
-    }
-
-    if ($this->shouldDowngradePhpunit()) {
-      $additions[] = 'phpunit/phpunit:9.4.3';
-    }
```

Comment:
Let's move `phpspec/prophecy-phpunit:^2` to `packages.yml` . 

But the condition to down grade phpunit mentioned in the function `shouldDowngradePhpunit` will always be false. So this may be removed.